src/agent/main.cpp

```diff
@@ -230,7 +232,11 @@ void otPlatReset(otInstance *aInstance)
 
     gPlatResetReason = OT_PLAT_RESET_REASON_SOFTWARE;
 
-    otSysDeinit();
```

Comment:
Why remove the call to `otSysDeinit()`?

Reply:
`Application::Deinit()` calls `ControllerOpenThread::Deinit()` which calls `otSysDeinit()`.